.github/workflows/release.yml

```diff
@@ -0,0 +1,133 @@
+name: Release
+
+on:
+  # pull_request:
+    # branches: [ master ]
+  push:
+    tags:
+      - "*beta*"
+      - "*pre*"
+      # - "1[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  prerelease:
+    name: Pre-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout highlight.js
+        uses: actions/checkout@v2
+
+      - name: Tag is ${{ github.ref }}.
+        # we have to repeat ourselves here since the environment is not actually updated
+        # until the next step executes, so we can't access $TAG_NAME yet
+        run: |
+          echo "TAG_NAME=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+          echo "MAJOR_VERSION=$(echo ${GITHUB_REF/refs\/tags\//} | cut -d'.' -f1)" >> $GITHUB_ENV
+      - name: Make sure we are pushing a tag...
```

Comment:
Debugging purposes or when would this workflow run when not a tag?

Reply:
It’s just a fail safe.